app_dart/lib/src/request_handlers/push_build_status_to_github.dart

```diff
@@ -73,7 +73,11 @@ class PushBuildStatusToGithub extends ApiRequestHandler<Body> {
     final Map<LuciBuilder, List<LuciTask>> luciTasks = await luciService.getRecentTasks(builders: postsubmitBuilders);
 
     String status = GithubBuildStatusUpdate.statusSuccess;
-    for (List<LuciTask> tasks in luciTasks.values) {
+    for (LuciBuilder luciBuilder in luciTasks.keys) {
+      final List<LuciTask> tasks = luciTasks[luciBuilder];
```

Comment:
Avoid the value lookup, iterate over the keys and values at the same time.
```suggestion
    luciTasks.forEach((LuciBuilder luciBuilder, List<LuciTask> tasks) {
```

Reply:
Thanks. Updated.